pages/[slug].js

```diff
@@ -0,0 +1,49 @@
+import { useRouter } from "next/router";
+import Head from "next/head";
+import Image from "next/image";
+import styled from "styled-components";
+import { StyledOutivityImage } from "@/components/OutivityImage/StyledOutivityImage";
```

Comment:
As mentioned above, naming the file of the image component `index.js` would also shorten your import here.